File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n, i, j, k;
    do
    {
        n = get_int("Hight: ");

    }
    while (n <= 0 || 8 < n);

    for (i = 1; i <= n; i++)
    {
        for (j = 0; j < n - i; j++)
        {
            printf(" ");
        }
        for (j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }


}

File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if 0 < n and n <= 8:
        for i in range(1, n + 1):
            print(" " * (n - i), end="")
            print("#" * i, end="  ")
            print("#" * i)

        break
